packages/output/src/browser/output-widget.tsx

```diff
@@ -78,12 +78,31 @@ export class OutputWidget extends ReactWidget {
         </React.Fragment>;
     }
 
-    public clear(): void {
+    clear(): void {
         if (this.outputChannelManager.selectedChannel) {
             this.outputChannelManager.selectedChannel.clear();
         }
     }
 
+    selectAll(): void {
+        if (this.outputChannelManager.selectedChannel) {
+            const element = document.getElementById(OutputWidget.IDs.CONTENTS);
+            if (element) {
+                const selectElementContent = (what: HTMLElement) => {
```

Comment:
[minor] perhaps we can find a better name instead of `what`.

Reply:
What do you suggest?

Reply:
> What do you suggest?

I'm not sure, either `content` or even `htmlElement`?